app/src/interfaces/_system/system-permissions/system-permissions.vue

```diff
@@ -570,117 +569,96 @@ function useGroupedPermissions() {
-					:permissions="group.permissions"
-					:collection="group.collection"
-					@edit-item="editItem(group.collection.collection, $event)"
-					@remove-row="removeCollection(group.collection.collection)"
-					@set-full-access-all="setFullAccessAll(group.collection.collection)"
-					@set-no-access-all="setNoAccessAll(group.collection.collection)"
-					@set-full-access="setFullAccess(group.collection.collection, $event)"
-					@set-no-access="setNoAccess(group.collection.collection, $event)"
-				/>
-
-				<tr>
-					<td colspan="7" class="system-divider">
-						<v-divider v-if="regularPermissions.length > 0 && systemPermissions.length > 0">
-							{{ t('system_collections') }}
-						</v-divider>
-					</td>
-				</tr>
-
-				<permissions-row
-					v-for="group in systemPermissions"
-					:key="group.collection.collection"
-					:permissions="group.permissions"
-					:collection="group.collection"
-					:policy="primaryKey"
-					:disabled-actions="disabledActions[group.collection.collection]"
-					:app-minimal="
-						appAccess
-							? appAccessMinimalPermissions.filter(
-									(permission) => permission.collection === group.collection.collection,
-							  )
-							: undefined
-					"
-					@edit-item="editItem(group.collection.collection, $event)"
-					@remove-row="removeCollection(group.collection.collection)"
-					@set-full-access-all="setFullAccessAll(group.collection.collection)"
-					@set-no-access-all="setNoAccessAll(group.collection.collection)"
-					@set-full-access="setFullAccess(group.collection.collection, $event)"
-					@set-no-access="setNoAccess(group.collection.collection, $event)"
-				/>
-
-				<tr v-if="appAccess">
-					<td colspan="7" class="reset-toggle">
-						<span>
-							{{ t('reset_system_permissions_to') }}
-							<button @click="resetActive = 'minimum'">{{ t('app_access_minimum') }}</button>
-							/
-							<button @click="resetActive = 'recommended'">{{ t('recommended_defaults') }}</button>
-						</span>
-					</td>
-				</tr>
-
-				<add-collection-row
-					:exclude-collections="
-						[...regularPermissions, ...systemPermissions].map(({ collection }) => collection.collection)
-					"
-					@select="addEmptyPermission($event)"
-				/>
-			</tbody>
-		</table>
-	</div>
 
-	<permissions-detail
-		:active="currentlyEditingKey !== null"
-		:edits="editsAtStart"
-		:permission-key="currentlyEditingKey"
-		:policy-key="primaryKey"
-		:policy-edits="values"
-		@input="stageEdits"
-		@update:active="cancelEdit"
-	/>
-
-	<v-dialog :model-value="!!resetActive" @update:model-value="resetActive = false" @esc="resetActive = false">
-		<v-card>
-			<v-card-title>
-				{{ t('reset_system_permissions_copy') }}
-			</v-card-title>
-			<v-card-actions>
-				<v-button secondary @click="resetActive = false">{{ t('cancel') }}</v-button>
-				<v-button @click="resetSystemPermissions(resetActive === 'recommended')">
-					{{ t('reset') }}
-				</v-button>
-			</v-card-actions>
-		</v-card>
-	</v-dialog>
```

Comment:
Why was this removed?

Reply:
![dontknow](https://github.com/user-attachments/assets/5f2f53c4-6351-4c5a-838c-af9a9934dd8e)